src/openrct2-ui/windows/Ride.cpp

```diff
@@ -781,7 +781,7 @@ static std::unique_ptr<TrackDesign> _trackDesign;
 // Cached overall view for each ride
 // (Re)calculated when the ride window is opened
 struct ride_overall_view {
-    int16_t x, y, z;
+    int32_t x, y, z;
```

Comment:
Can be backported.
Better yet, changed to CoordsXYZ or TileCoordsXYZ.

Reply:
yeah doing that atm.